ftp/1.6/modules/ROOT/pages/ftp-studio-configuration.adoc

```diff
@@ -0,0 +1,124 @@
+= Using Anypoint Studio to Configure FTP Connector 1.6 - Mule 4
+
+Anypoint Studio (Studio) editors help you design and update your Mule applications, properties, and configuration files.
+
+To add and configure a connector in Studio:
+
+. <<create-mule-project,Create a Mule project>>.
+. <<add-connector-to-project,Add the connector to your Mule project>>.
+. <<configure-input-source,Configure a source for the connector's flow>>.
+. <<add-connector-operation,Add a connector operation to the flow>>.
+. <<configure-global-element,Configure a global element for the connector>>.
+
+When you run the connector, you can view the app log to check for problems in real time, as described in <<view-app-log,View the App Log>>.
+
+If you are new to configuring connectors in Studio, see xref:connectors::introduction/intro-config-use-studio.adoc[Use Studio to Configure a Connector]. If, after reading this topic, you need additional information about the connector fields, see the xref:ftp-documentation.adoc[FTP Connector Reference Reference].
```

Comment:
I recommend revising the second sentence like this:
"If you need additional information about the connector fields after reading this topic, see the xref:ftp-documentation.adoc[FTP Connector Reference Reference]."